frontend/src/app/home/homepage.component.ts

```diff
@@ -1,10 +1,26 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
 
-import { OffersComponent } from '../offers/offers.component';
+import { OffersService } from 'app/homepage-offer/offers.service';
+import { Offer } from 'app/homepage-offer/offers.model';
 
 @Component({
   selector: 'volontulo-home',
   templateUrl: './homepage.component.html',
-  styleUrls: ['./homepage.component.css']
+  styleUrls: ['./homepage.component.css'],
```

Comment:
this css is empty, so:
 * delete the file
 * remove this line

Reply:
We added some style to this file so it should be state in solution.